doc/source/train/user_guide.rst

```diff
@@ -319,19 +319,19 @@ Each ``Trainer`` will have a local directory created for logs, and each call
 to ``Trainer.run`` will create its own sub-directory of logs.
 
 By default, the ``logdir`` will be created at
-``~/ray_results/train_<datestring>``.
+``./ray_results/train_<datestring>``.
 This can be overridden in the ``Trainer`` constructor to an absolute path or
-a path relative to ``~/ray_results``.
+a path relative to ``./ray_results``.
```

Comment:
For this I think we should keep the default behavior the same (`~/ray_results/train_<datestring>`). Only when the user specifies a path should we use the relative path.

Examples:
1. User does not specify anything: `~/ray_results/train_<datestring>`
2. User specifies an absolute path: `<abs_path>`
3. User specifies a relative path: `<cwd>/<relative_path>`

Reply:
Secondly, when the user specifies the path we do not need to prepend `ray_results`.

Reply:
Got it.